File: src/net/eduard/api/command/api/ApiWorldsCommand.java

package net.eduard.api.command.api;

import org.bukkit.Bukkit;
import org.bukkit.World;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class ApiWorldsCommand extends CommandManager {

	public ApiWorldsCommand() {
		super("worlds","mundos");
		setDescription("Mostra uma lista de mundos carregados no servidor");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		StringBuilder mundos = new StringBuilder();

		for (World world : Bukkit.getWorlds()) {
			if (mundos.length() > 0) {
				mundos.append("§a, §f");
			}
			mundos.append("" + world.getName());
		}
		sender.sendMessage("§aOs mundo existentes s§o: §f" + mundos.toString());

		return true;
	}

}

File: src/main/java/net/eduard/api/command/api/ApiWorldsCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import java.lang.StringBuilder

class ApiWorldsCommand : CommandManager("worlds", "mundos","worldlist") {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        val mundos = StringBuilder()
        for (world in Bukkit.getWorlds()) {
            if (mundos.isNotEmpty()) {
                mundos.append("§a, §f")
            }
            mundos.append("" + world.name)
        }
        sender.sendMessage("§bEduardAPI §aOs mundo existentes são: §f$mundos")
        return true
    }

    init {
        description = "Mostra uma lista de mundos carregados no servidor"
    }
}